plugins/lsp/lspsaga.nix

```diff
@@ -7,220 +7,651 @@
+
+          sign = helpers.defaultNullOpts.mkBool true "Show sign in status column.";
+
+          debounce = helpers.defaultNullOpts.mkInt 10 "Timer debounce.";
+
+          signPriority = helpers.defaultNullOpts.mkInt 40 "Sign priority.";
+
+          virtualText = helpers.defaultNullOpts.mkBool true "Show virtual text at the end of line.";
+
+          enableInInsert = helpers.defaultNullOpts.mkBool true "Enable in insert.";
         };
 
-        info = mkOption {
-          type = types.nullOr types.str;
-          default = "  Info";
-          description = "Info diagnostic header";
+        scrollPreview = {
+          scrollDown = mkKeymapOption "<C-f>" "Scroll down.";
+
+          scrollUp = mkKeymapOption "<C-b>" "Scroll up.";
         };
-      };
 
-      maxDialogWidth = mkOption {
-        type = types.nullOr types.int;
-        default = null;
-        description = "Maximum dialog width";
-      };
+        requestTimeout = helpers.defaultNullOpts.mkInt 2000 "Request timeout.";
+
+        finder = {
+          maxHeight = helpers.defaultNullOpts.mkNullable (types.numbers.between 0.0 1.0) "0.5" ''
+            `max_height` of the finder window (float layout).
+          '';
+
+          leftWidth = helpers.defaultNullOpts.mkNullable (types.numbers.between 0.0 1.0) "0.3" ''
+            Width of the left finder window (float layout).
+          '';
+
+          rightWidth = helpers.defaultNullOpts.mkNullable (types.numbers.between 0.0 1.0) "0.3" ''
+            Width of the right finder window (float layout).
+          '';
+
+          methods = helpers.defaultNullOpts.mkNullable (with types; attrsOf str) "{}" ''
+            Keys are alias of LSP methods.
+            Values are LSP methods, which you want show in finder.
 
-      icons = {
-        codeAction = mkOption {
-          type = types.nullOr types.str;
-          default = null;
-          description = "Code action icon";
+            Example:
+            ```nix
+              {
+                tyd = "textDocument/typeDefinition";
+              }
+            ```
+          '';
+
+          default = helpers.defaultNullOpts.mkStr "ref+imp" ''
+            Default search results shown, `ref` for "references" and `imp` for "implementation".
+          '';
+
+          layout = helpers.defaultNullOpts.mkEnumFirstDefault ["float" "normal"] ''
+            `normal` will use the normal layout window priority is lower than command layout.
+          '';
+
+          silent = helpers.defaultNullOpts.mkBool false ''
+            If it’s true, it will disable show the no response message.
+          '';
+
+          filter = helpers.defaultNullOpts.mkNullable (with types; attrsOf str) "{}" ''
+            Keys are LSP methods.
+            Values are a filter handler.
+            Function parameter are `client_id` and `result`.
+          '';
+
+          spInexist = helpers.defaultNullOpts.mkBool false "";
```

Comment:
This does not appear to be documented in https://nvimdev.github.io/lspsaga/finder/

Reply:
removed